fakebook/src/app/services/follow.service.ts

```diff
@@ -9,31 +9,30 @@ import { OktaAuthService } from '@okta/okta-angular';
 })
 export class FollowService {
   constructor(private http: HttpClient, private oktaAuth: OktaAuthService) {}
-  url = `${environment.baseUrls.posts}/api`;
+  url = `${environment.baseUrls.profile}/api/profiles`;
 
-  follow(follower: User, followee: User): any {
+  follow(follower: string): any {
     const accessToken = this.oktaAuth.getAccessToken();
     const headers = {
       Authorization: 'Bearer ' + accessToken,
       Accept: 'application/json',
       'Content-Type': 'application/json',
     };
     return this.http
-      .post(`${this.url}/follows/${follower.id}`, null, { headers })
+      .post(`${this.url}/follow/${follower}`, null, { headers })
       .toPromise();
   }
 
-  unfollow(follower: User, followee: User): any {
+  unfollow(follower: string): any {
     const accessToken = this.oktaAuth.getAccessToken();
     const headers = {
       Authorization: 'Bearer ' + accessToken,
       Accept: 'application/json',
       'Content-Type': 'application/json',
     };
     return this.http
-      .delete(`${this.url}/follows/${follower.id}`, { headers })
-      .toPromise()
-      .then((res) => console.log(JSON.stringify(res)));
+      .post(`${this.url}/unfollow/${follower}`, null, { headers })
+      .toPromise();
```

Comment:
this is bad, we shouldn't be doing promise and `any` type, but i'll let it through